poetry/repositories/legacy_repository.py

```diff
@@ -444,6 +443,3 @@ def _get(self, endpoint: str) -> Optional[Page]:
             )
 
         return Page(response.url, response.content, response.headers)
-
-    def _download(self, url, dest):  # type: (str, str) -> None
-        return download_file(url, dest, session=self.session)
```

Comment:
Removed this, as it is an exact copy of the one from the base class, `PypiRepository`.